PeriodogramAnalysis/test_multitaper_spectrum_statistics.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from mne.time_frequency import psd_array_multitaper
+from miv.datasets.openephys_sample import load_data
+from miv.core.operator import Operator, DataLoader
+from miv.io.openephys import Data, DataManager
+from multitaper_spectrum_statistics import multitaper_psd
+
+path: str = load_data(progbar_disable=True).data_collection_path
```

Comment:
Don't define variable outside the scope, especially for testing

Reply:
I see, moved it in this commit [f5695ac]